baseplate/experiments/providers/simple_experiment.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import hashlib
+import time
+
+from .base import Experiment
+from ..._compat import long, iteritems
+
+
+logger = logging.getLogger(__name__)
+
+
+class SimpleExperiment(Experiment):
+    """Base class for simple experiments. Note that this class shares a lot of
+    code from the r2.py class, which is slated for deprecation and removal.
+    """
+
+    def make_seed(self, seed_data):
+        id = seed_data.get('id')
+        name = seed_data.get('name')
+        version = seed_data.get('version')
```

Comment:
version should NOT be used in the randomization.  Shuffle_version should be, although it's fine to exclude it for now if you want.

Reply:
oh, hmm.  Since this is legacy, maybe that's OK?  Still think it should not be, for consistency.

Reply:
note that the "version" used in this case is whatever version is desired to be used for seed generation. I actually pass the "shuffle_version" value in here, since that's the specific type of version we want to use for making the seed. So it's actually doing the right thing, but I agree that it's actually a bit convoluted. Will clean this up a bit.